service/src/LevelZero.cpp

```diff
@@ -187,7 +187,7 @@ namespace geopm
                             geopm::LevelZero::M_DOMAIN_SIZE);
 
             for (auto handle : freq_domain) {
-                zes_freq_properties_t property;
+                zes_freq_properties_t property = {};
```

Comment:
I see other instances of this pattern for the other property structs.
E.g. on line 67:
```
ze_device_properties_t property;
```
I think we should initialize all the property structs to zero.